sentry_sdk/scope.py

```diff
@@ -62,9 +65,20 @@
     T = TypeVar("T")
 
 
+_global_scope = None  # type: Optional[Scope]
+_isolation_scope = ContextVar("isolation_scope", default=None)
+_current_scope = ContextVar("current_scope", default=None)
```

Comment:
Might be helpful to add some kind of doc comments here to summarize the difference between current and isolation scopes

Reply:
done